src/main/resources/db/migration/V1_0_6__update.sql

```diff
@@ -0,0 +1,29 @@
+-- adding tables for multi users access
+
+CREATE TABLE `webusers` (
+  `username` varchar(50) COLLATE utf8mb3_unicode_ci NOT NULL,
+  `password` varchar(255) COLLATE utf8mb3_unicode_ci NOT NULL,
+  `enabled` tinyint(1) NOT NULL,
+  PRIMARY KEY (`username`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3 COLLATE=utf8mb3_unicode_ci;
+
+CREATE TABLE `webauthorities` (
+  `username` varchar(50) COLLATE utf8mb3_unicode_ci NOT NULL,
+  `authority` varchar(50) COLLATE utf8mb3_unicode_ci NOT NULL DEFAULT 'ROLE_USER',
+  UNIQUE KEY `authorities_idx_1` (`username`,`authority`),
+  CONSTRAINT `authorities_ibfk_1` FOREIGN KEY (`username`) REFERENCES `webusers` (`username`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3 COLLATE=utf8mb3_unicode_ci;
+
+CREATE UNIQUE INDEX ix_auth_username
+  on webauthorities (username,authority);
+
+-- Insert a user = admin with the password = pass. Change password after installing!
+INSERT INTO webusers (username, password, enabled)
+  values ('admin',
+    '$2a$10$.Rxx4JnuX8OGJTIOCXn76euuB3dIGHHrkX9tswYt9ECKjAGyms30W',

```

Comment:
current standard password is 1234. i would suggest to pass this value through the encoder and save it.

even better: we should probably think about a migration strategy of existing installations, such as...

we can read the `CONFIG.getAuth().getUserName()` and `CONFIG.getAuth().getEncodedPassword()` and insert this user into table when during boot, if the table does not have this entry already.

Reply:
postConstruct impl in SecurityConfiguration of https://github.com/steve-community/steve/pull/1539/commits/3491594b7b4898cda5d41c883cb76a8fa688a77a does exactly this.